data/mapObjects/blueshouse.asm

```diff
@@ -8,9 +8,9 @@ BluesHouseObject:
 	db 0 ; signs
 
 	db 3 ; objects
-	object SPRITE_DAISY, 2, 3, STAY, RIGHT, 1 ; Daisy, sitting by map
-	object SPRITE_DAISY, 6, 4, WALK, 1, 2, 0 ; Daisy, walking around
-	object SPRITE_BOOK_MAP_DEX, 3, 3, STAY, NONE, 3, 0 ; map on table
+	object SPRITE_DAISY, 2, 3, STAY, RIGHT, 129, 1 ; Daisy, sitting by map
```

Comment:
I think you misunderstood my previous comment. I was pointing out that you accidentally deleted a `, 0` argument at the end of this line.
You can undo this new change. The other numbers where correct the way they were.

Reply:
This may take time to fix. Please understand. Same thing with the other maps you pointed out.